pkg/service/v0/service.go

```diff
@@ -43,8 +43,92 @@ func New(cfg *config.Config) (s *Service, err error) {
 		return nil, err
 	}
 
-	if err = os.MkdirAll(filepath.Join(cfg.Server.AccountsDataPath, "accounts"), 0700); err != nil {
-		return nil, err
+	// check if accounts exist
+	accountsDir := filepath.Join(cfg.Server.AccountsDataPath, "accounts")
+	var fi os.FileInfo
+	if fi, err = os.Stat(accountsDir); err != nil {
+		if os.IsNotExist(err) {
+			// create accounts directory
+			if err = os.MkdirAll(accountsDir, 0700); err != nil {
```

Comment:
`0644` to prevent spooky things

Reply:
nope, this is a dir. needs executable bit ... and noone but this service should have access